wandb/internal/sender.py

```diff
@@ -311,6 +343,16 @@ def handle_artifact(self, data):
             use_after_commit=artifact.use_after_commit,
         )
 
+    def handle_get_summary(self, data):
+        resp = wandb_internal_pb2.ResultRecord()
+        for key, value in six.iteritems(self._consolidated_summary):
+            item = wandb_internal_pb2.SummaryItem()
+            item.key = key
```

Comment:
so, my understanding is that given the recursion there's going to be in the summary implementation to unpack these values, we _don't_ need to recurse or unpack them here. Is that correct?